src/openrct2-ui/windows/SavePrompt.cpp

```diff
@@ -159,7 +160,7 @@ class SavePromptWindow final : public Window
                 {
                     intent = std::make_unique<Intent>(WindowClass::Loadsave);
                     intent->PutExtra(INTENT_EXTRA_LOADSAVE_TYPE, LOADSAVETYPE_SAVE | LOADSAVETYPE_LANDSCAPE);
-                    intent->PutExtra(INTENT_EXTRA_PATH, gScenarioName);
+                    intent->PutExtra(INTENT_EXTRA_PATH, OpenRCT2::GetGameState().ScenarioName);
```

Comment:
Could you add `using namespace OpenRCT2;` under the includes at the top, and dorp the `OpenRCT2::` part here?